app/core/Router.coffee

```diff
@@ -157,8 +157,12 @@ module.exports = class CocoRouter extends Backbone.Router
     'play/ladder/:levelID': go('ladder/LadderView')
     'play/ladder': go('ladder/MainLadderView')
     'play/level/:levelID': go('play/level/PlayLevelView')
-    'play/game-dev-level/:levelID/:sessionID': go('play/level/PlayGameDevLevelView')
-    'play/web-dev-level/:levelID/:sessionID': go('play/level/PlayWebDevLevelView')
+    'play/game-dev-level/:sessionID': go('play/level/PlayGameDevLevelView')
+    'play/web-dev-level/:sessionID': go('play/level/PlayWebDevLevelView')
+    'play/game-dev-level/:levelID/:sessionID': (levelID, sessionID) ->
+      @navigate("play/game-dev-level/#{sessionID}", { trigger: true, replace: true })
+    'play/web-dev-level/:levelID/:sessionID': (levelID, sessionID) ->
+      @navigate("play/web-dev-level/#{sessionID}", { trigger: true, replace: true })
```

Comment:
I was originally imagining the url would stay the same, just that you could have `levelID` be either a slug or the original. I guess, though, this would work better because now any broken links will be rerouted and work. This seems fine. But shouldn't the share urls we give them be updated to match?

Reply:
Whoops! Yes, I will update the links we show them.